terraform/aws/modules/get_os_image/main.tf

```diff
@@ -1,11 +1,12 @@
 # Data used to get the correct AMI image
 data "aws_ami" "image" {
-  most_recent = true
-  owners      = [var.os_owner]
+  most_recent        = true
+  owners             = [var.os_owner]
+  include_deprecated = true
 
   filter {
     name   = "name"
-    values = substr(var.os_image, 0, 4) == "ami-" ? ["*"] : ["${var.os_image}-*"]
```

Comment:
I suggest to move this change in a dedicated PR

Reply:
Thanks for the comments. I recovered the code.